tests/conditional_compilation/conftest.py

```diff
@@ -171,3 +190,63 @@ def save_session_info(pytestconfig, artifacts):
     """Fixture function for saving additional attributes to configuration file."""
     yield
     write_session_info(path=artifacts / SESSION_INFO_FILE, data=pytestconfig.session_info)
+
+
+@pytest.fixture(scope="function")
+def prepare_models(openvino_ref, models, request):
+    """
+    Process models: prepare Open Model Zoo models, skip non-OMZ models.
+    """
+    for model in models:
+        if "OMZ" in model.get("type", "TESTDATA").upper():
```

Comment:
1. no need to specify default for `model.get`
2. I would prefer to see only environment vars in uppercase - it's more readable (the note may be ignored)